app/lib/getNearbyServices.js

```diff
@@ -1,23 +1,71 @@
-const log = require('../../app/lib/logger');
+require('moment-timezone');
+
+const timezone = require('../../config/config').timezone;
+const log = require('../lib/logger');
 const esClient = require('./esClient');
-const filterServices = require('./filterServices');
+const addMessages = require('./addMessages');
+const getDateTime = require('./getDateTime');
 const VError = require('verror').VError;
-const esGetPharmacyHistogram = require('./promHistograms').esGetPharmacy;
+const esGetNearbyPharmacyHistogram = require('./promHistograms').esGetNearbyPharmacy;
+const esGetOpenPharmacyHistogram = require('../lib/promHistograms').esGetOpenPharmacy;
```

Comment:
The navigation in the require can be simplified in the same way it is in the line above.